alchemist-elixir-ls.el

```diff
@@ -55,14 +50,10 @@
     project-erlang-version))
 
 (defun alchemist--server-extension ()
-  (let ((extension (or alchemist-server-extension
-                       (completing-read "Choose the kind of executable that runs on this system: "
-                                        '("sh" "bat")
-                                        nil
-                                        t
-                                        ))))
-    (setq alchemist-server-extension extension)
-    extension))
+  (defvar alchemist-system-extension)
```

Comment:
If the variable is no longer public-facing then we should call it `alchemist--system-extension` to indicate that it is private.

Reply:
Sure.